com.milesalan.mepo.yml

```
@@ -0,0 +1,123 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ziglang
+rename-icon: mepo
+rename-desktop-file: mepo.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-documents
+
+build-options:
+  append-path: /usr/lib/sdk/ziglang
+command: mepo
+modules:
+  # Core build deps for mepo (note SDL2 is in runtime)
+  - name: SDL2_image
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_image/releases/download/release-2.6.2/SDL2_image-2.6.2.tar.gz
+        sha256: 48355fb4d8d00bac639cd1c4f4a7661c4afef2c212af60b340e06b7059814777
+
+  - name: SDL2_ttf
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.20.1/SDL2_ttf-2.20.1.tar.gz
+        sha256: 78cdad51f3cc3ada6932b1bb6e914b33798ab970a1e817763f22ddbfd97d0c57
+
+  - name: SDL2_gfx
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.4.tar.gz
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+      - type: shell
+        commands:
+          - autoreconf -fvi
```

```suggestion
          - cp -p /usr/share/automake-*/config.{sub,guess} .
          - autoreconf -fvi
```

You probably need that too.

---

Fixed